source/ranges.tex

```diff
@@ -6138,6 +6146,7 @@
 
   private:
     template<bool> struct @\exposid{iterator}@;                     // \expos
+    template<bool> struct @\exposid{sentinel}@;                     // \expos
```

Comment:
We've been adding cross-links to the definitions of these nested types in other ranges:
```suggestion
    // \ref{range.elements.sentinel}, class template \tcode{elements_view::\exposid{sentinel}}
    template<bool> struct @\exposid{sentinel}@;                     // \expos
```
(#3750 adds the similar cross-reference for `iterator`.)